ci-operator/config/openshift/lightspeed-service/openshift-lightspeed-service-main.yaml

```diff
@@ -9,6 +9,9 @@ images:
 - dockerfile_path: Containerfile.rag
   from: lightspeed-service-api
   to: lightspeed-service-api-rag
+- dockerfile_path: Containerfile.rag.test
+  from: lightspeed-service-api-rag
+  to: lightspeed-service-api-rag-test
```

Comment:
why do we need to introduce this image instead of continuing to use the source image?

at some point we might need it if it we want to run these tests against PRs in another repo, but you're not doing that here

Reply:
Because the e2e tests need both RAG and the tests. I looked for a way to mount the rag-content image to the src image, but didn't find a way to use it.